app/models/pusher.rb

```diff
@@ -119,7 +119,7 @@ def after_write
       Mailer.delay.gem_pushed(user.id, @version_id, notified_user.id)
     end
     Delayed::Job.enqueue Indexer.new, priority: PRIORITIES[:push]
-    rubygem.delay.index_document
+    rubygem.delay.reindex
```

Comment:
fwiw, `delay.reindex` isn't tested in Searchkick (the recommended approach is `reindex(mode: :async)`)

Reply:
This just calls `rubygem.reindex` in backround job. That should be safe.

Reply:
We don't use ActiveJob for now, but I have plan to migrate from delay job being used directly to ActiveJob. Once done, we can move to `reindex(mode: :async)` if I understand it well.